### PR TITLE
Export format errors utils in embedding utils

### PR DESCRIPTION
**Pull request by helen-dikareva**

### CODE REVIEW

**Comment by AndreyBelym** [comment]
The test failure in Firefox is not related to the changes, please review it as is.

File: src/errors/test-run/utils.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    [TEST_RUN_PHASE.initial]:                 '',
+    [TEST_RUN_PHASE.inFixtureBeforeHook]:     '<span class="subtitle">Error in fixture.before hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureBeforeEachHook]: '<span class="subtitle">Error in fixture.beforeEach hook</span>\n',
+    [TEST_RUN_PHASE.inTestBeforeHook]:        '<span class="subtitle">Error in test.before hook</span>\n',
+    [TEST_RUN_PHASE.inTest]:                  '',
+    [TEST_RUN_PHASE.inTestAfterHook]:         '<span class="subtitle">Error in test.after hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureAfterEachHook]:  '<span class="subtitle">Error in fixture.afterEach hook</span>\n',
+    [TEST_RUN_PHASE.inFixtureAfterHook]:      '<span class="subtitle">Error in fixture.after hook</span>\n',
+    [TEST_RUN_PHASE.inRoleInitializer]:       '<span class="subtitle">Error in Role initializer</span>\n',
+    [TEST_RUN_PHASE.inBookmarkRestore]:       '<span class="subtitle">Error while restoring configuration after Role switch</span>\n'
```
Comment (AlexKamaev) [inline src/errors/test-run/utils.js]: Should not we remove '<span class=.....' from these constants?

Comment (helen-dikareva) [reply src/errors/test-run/utils.js]: nop, they are ok

**Comment by helen-dikareva** [comment]
Can we merge this PR? It is obvious that there are blinking tests?

**Pull request merged**